django_netjsonconfig/base/admin.py

```diff
@@ -200,6 +204,24 @@ def download_view(self, request, pk):
         return send_file(filename='{0}.tar.gz'.format(config.name),
                          contents=config_archive.getvalue())
 
+    def device_context_view(self, request, pk):
+        instance = get_object_or_404(self.model, pk=pk)
+        authenticated = request.user.is_authenticated
+        if callable(authenticated):
+            authenticated = authenticated()
```

Comment:
all these authentication checks will be done by `self.admin_site.admin_view`, once you add that call you should be able to remove this logic